packages/web-react/src/components/Header/HeaderDesktopActions.tsx

```diff
@@ -3,12 +3,11 @@ import classNames from 'classnames';
 import { useStyleProps } from '../../hooks';
 import { HeaderDesktopActionsProps } from '../../types';
 import { useHeaderStyleProps } from './useHeaderStyleProps';
-import { HEADER_ACTIONS_COLOR_DEFAULT } from './constants';
 
 const HeaderDesktopActions = (props: HeaderDesktopActionsProps) => {
-  const { color = HEADER_ACTIONS_COLOR_DEFAULT, ...restProps } = props;
+  const { isAtEnd = false, ...restProps } = props;
```

Comment:
in other react components we are defining default values above component like this:
```
const defaultProps = {
  isAtEnd: false,
};
```

and then in the component we do:
```
const propsWithDefaults = { ...defaultProps, ...props };
const { isAtEnd, ...restProps } = propsWithDefaults;
```